tests/golem/verificator/test_core_verifier.py

```diff
@@ -51,3 +53,19 @@ def test_simple_verification(self):
         verification_data["results"] = ["not a file"]
         core_verifier.simple_verification(verification_data)
         assert core_verifier.state == SubtaskVerificationState.WRONG_ANSWER
+
+    @staticmethod
+    def test_task_timeout():
```

Comment:
just one thing, please add `#TODO: fix it`, not to forget (I can do it as a part of my current assignment)
```suggestion
    def test_task_timeout():  #TODO: fix it`
```

Reply:
:+1: done